src/index.js

```diff
@@ -233,13 +233,13 @@ export default class extends Component {
   }
 
   // include internals with state
-  fullState () {
+  fullState() {
     return Object.assign({}, this.state, this.internals)
   }
 
   onLayout = (event) => {
     const { width, height } = event.nativeEvent.layout
-    const offset = this.internals.offset = {}
+    const offset = this.internals.offset = { x: 0, y: 0 }
```

Comment:
You should be able to add it to internals var line 228, shouldn't you ?

Reply:
So I realize I'm... 2 years late; did we fix this? lol

Reply:
It doesn't work right. Should be so
`const offset = this.internals.offset = { x: width, y: height }`